bin/check-apache-status.rb

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env ruby
+#
+#   check-apache-status
+#
+# DESCRIPTION:
+#   Test whether apache is up and functional by using mod_status
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#
+# USAGE:
+# check-apache-status.rb --help
+# NOTES:
+#
+# LICENSE:
+# Magic Online http://www.magic.fr  @2016
+# Released under the same terms as Sensu (the MIT license); see LICENSE  for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'open3'
+require 'net/http'
+require 'net/https'
+
+#
+# Check  Apache Status
+#
+class CheckApacheStatus < Sensu::Plugin::Check::CLI
+  option :busy, long: '--busy BUSYWORKERS', short: '-b BusyWorkers', description: 'Busy workers threshold', proc: proc(&:to_i), default: 150, required: false
```

Comment:
could you split these options into multiple lines? This is a bit hard to read, something like:
```ruby
  option :busy,
         long: '--busy BUSYWORKERS',
         short: '-b BusyWorkers',
         description: 'Busy workers threshold',
         proc: proc(&:to_i),
         default: 150,
         required: false
```